saged/disease_vector.py

```diff
@@ -0,0 +1,170 @@
+"""
+This script will jointly simulate data from healthy and disease data. It will then calculate
+the centroid of the disease samples in the embedded space and use that centroid to extrapolate
+datapoints to create easier and harder simulated classification problems
+"""
+
+import argparse
+import os
+import pickle
+import random
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+import yaml
+from keras import backend as K
+from ponyo import vae
+from sklearn import preprocessing
+
+from saged import datasets
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('dataset_config',
+                        help='The config file containing info about the dataset')
+    parser.add_argument('compendium_metadata',
+                        help='A json file containing information compendium_file')
+    parser.add_argument('out_dir', default='data/simulated',
+                        help='The directory to store the simulated data files in')
+
+    parser.add_argument('--simulation_config', default='model_configs/simulation/tybalt.yml')
+    parser.add_argument('--sample_count', default=1000, type=int,
+                        help='The number of samples to generate for each translated dataset')
+
+    parser.add_argument('--label',
```

Comment:
Should `args.label` and `args.negative_class` cover the whole dataset? Like in this case any of the input samples should be either sepsis or healthy (i.e. none of them are TB samples or any other non-healthy, non-sepsis condition)?

Reply:
They should cover the entire dataset, but since the arrays themselves aren't labeled we need to pass in the information so that the script knows what disease it's working with